reference/create-index-using-pgroonga.md

```diff
@@ -408,7 +408,8 @@ CREATE INDEX pgroonga_memos_index
                 NormalizerNFKC130,
                 NormalizerTable(
                   "normalized", "${table:pgrn_normalizations_index}.normalized",
-                  "target", "target"
+                  "target", "target",
+                  "report_source_offset", true
```

Comment:
This is not required for users who don't need `pgroonga_highlight_html()`.

Could you add one more example instead of changing this simple example?

And we also need `tokenizer=TokenNgram("report_source_location", true)` to `pgroonga_highlight_html()`. Only `report_source_offset` isn't enough.

Reply:
OK. I will make another example here. MyToDo++.